node/src/neighborhood/mod.rs

```diff
@@ -3681,6 +3710,11 @@ mod tests {
         let (call_database, call_agrs, call_gossip_source, neighborhood_metadata) =
             handle_params.remove(0);
         assert!(handle_params.is_empty());
+        subject_node.inner.country_code = call_database.root().inner.country_code.clone();
+        subject_node.metadata.node_location_opt =
+            call_database.root().metadata.node_location_opt.clone();
+        subject_node.metadata.last_update = call_database.root().metadata.last_update;
+        subject_node.resign();
```

Comment:
This is really fishy set of lines. I wish that you will conclude this is unnecessary. 

Going deeper in it, I must ask myself questions like: Why should the root node record change at all? The gossip was to be **Ignored** as the preparation of mocks shows at the top of the test. That means that the gossip contained only what we have already known. Thus no change to our database should be done.

I cannot think of a reason why you should allow these changes that you allow by the extra assertions. It doesn't seem right. 

Please defend yourself by sharing your understanding with me or try to take these lines back off.

Reply:
that is true, only thing is changed is last update. So I let only last update to change.